wasm_exec.js

```diff
@@ -238,7 +116,7 @@
       };
 
       const storeValue = (addr, v) => {
-        const nanHead = 0x7ff80000;
+        const nanHead = 0x7FF80000;
```

Comment:
In the process of trying to find a solution, I updated this with a newer version of tinygo's `wasm_exec.js`.  Because of that there's quite a few little changes like this.